docs/rules/jsx-sort-default-props.md

```diff
@@ -1,6 +1,8 @@
-# Enforce defaultProps declarations alphabetical sorting (react/jsx-sort-default-props)
+# Enforce defaultProps declarations alphabetical sorting (`react/jsx-sort-default-props`)
 
-❌ This rule is deprecated. Please use the [`sort-default-props`](./sort-default-props.md) rule instead.
+❌ This rule is deprecated. It was replaced by [sort-default-props](sort-default-props.md).
```

Comment:
```suggestion
❌ This rule is deprecated. It was replaced by [`sort-default-props`](sort-default-props.md).
```
since it seems like consistently backticking rule names is part of this tool?

Reply:
Fixed in this release: https://github.com/bmish/eslint-doc-generator/releases/tag/v0.14.0